apps/dashboard/src/modules/user/queries/use-user-get-query.ts

```diff
@@ -4,3 +4,17 @@ export const useUserGetQuery = (id: string) => {
   const { data: user, ...query } = trpc.user.get.useQuery(id)
   return { user, ...query }
 }
+
+export const useSearchUsersFromIDP = (searchQuery: string) => {
```

Comment:
Where the users come from is insignificant, I suggest we rename

```suggestion
export const useSearchUsers = (searchQuery: string) => {
```